ethcore/builtin/src/lib.rs

```diff
@@ -207,6 +221,118 @@ impl ModexpPricer {
 	}
 }
 
+/// Bls12 pairing price
+#[derive(Debug, Copy, Clone)]
+struct Bls12PairingPrice {
+	base: u64,
+	pair: u64,
+}
+
+/// bls12_pairing pricing model. This computes a price using a base cost and a cost per pair.
+#[derive(Debug)]
+struct Bls12PairingPricer {
+	price: Bls12PairingPrice,
+}
+
+/// Pricing for constant Bls12 operations (ADD and MUL in G1 and G2, as well as mappings)
+#[derive(Debug, Copy, Clone)]
+pub struct Bls12ConstOperationsPricer {
```

Comment:
```suggestion
pub struct Bls12ConstOperations {
```

Reply:
Does this mean `AltBn128PairingPricer` and `ModexpPricer` structs above are also incorrectly named? There appears to be an existing naming inconsistency somewhere.

Reply:
Technically, yes, but we have `AltBn128ConstOperations` so some "consistant inconsistency" is preferable.